_pages/en/esdc-it-strategy.md

```diff
@@ -52,43 +45,114 @@ But a complex and uncertain world rewards an empirical cycle of trying, observin
+## ESDC IT Strategy Logic Model
 
-The core of ESDC's IT Strategy focuses on reducing the risks associated with IT and increasing business agility.
-It will do so by
+ESDC's IT Strategy is not about trying to predict the future as it accepts that the future is highly complex and unpredictable. Instead, the [core of ESDC's IT Strategy](mandate.html) focuses on reducing the risks associated with IT and increasing business agility.
+It will do so by:
 
-- Changing the relationship service lines have with IT from a contracting model one to a partnership one:
-  - IITB will have a shared accountability over ESDC's service performance
-  - ESDC programs will have a shared accountability over the costs and risks of using IT as part of their service delivery
-- Increasing the rate at which IT is able to respond to change and deliver services
+- moving to a shared accountability amongst software stakeholders (which includes "the business")
+- improving the responsiveness of IT by investing in internal capabilities
 
-### IM/IT Services and key functions
+Achieving the above is mostly about cultural change as the ESDC IT Strategy looks to empower its existing teams, giving them more autonomy so they can more quickly respond to change.
+Hence the strategy will focus heavily on improving the culture within the organization.
+It will do so in an [iterative fashion](https://cloud.google.com/solutions/devops/devops-culture-transform), making course corrections as need be.
 
-The following diagram highlights IM/IT's services and key functions as represented in the [Senior Leadership Meeting on IM/IT](http://dialogue/grp/BU6810070/Shared%20Documents/Reference%20Materials/ESDC%20IMIT%20Moving%20Forward%20(DM%20Meeting).pptx).
+Because strategy making fosters long term impacts which are difficult to measure directly, the following IT Strategy Logic model helps visualize the causes and effects of output activities that are expected to directly affect short term outcomes and lead to long term impacts.
 
-![IM/IT services and key functions](assets/images/imit-services-and-functions.png "IM/IT Services and key functions")
+![ESDC's IT Strategy Logic Model - showing the short, medium, and long term outcomes](assets/images/it-strategy-logic-model.png "IT Strategy Logic Model")
 
-The goal of the ESDC IT Strategy is to improve each of those services and functions without telling their respective teams *what to do*.
-Instead, the IT Strategy focuses on empowering the teams to continuously improve by **investing in core competencies**.
+The above diagram positions ESDC's IT within its current state of transitioning to Digital Government at the same time as requiring to maintain its legacy systems until they are being replaced with modern ones. In addition, the [4 high level priorities of IITB](http://esdc.prv/en/iitb/corporate/Who_We_Are/Plans_and_Priorities/index.shtml) are listed to remind the focus of the organization.
 
-### Empowering IT teams
```

Comment:
Capitalization, teams should be capitalized

Reply:
that line has been removed :)